### ISSUE TITLE
httpDevServer is undefined

### ISSUE THREAD

**Issue by nitedani** [cyco130/vavite#8]
The vite-plugin-ssr example is broken on my computer.
Tested with Vavite 1.4.0, Node 16 and 18, Windows 11, WSL2.

1.3.x is working

```
import httpDevServer from "vavite/http-dev-server";
console.log(httpDevServer)  -> undefined
```

The error it causes:
pnpm run dev
```
    __vite_ssr_import_2__.default.on("request", app);
                                  ^

TypeError: Cannot read properties of undefined (reading 'on')
```

**Comment by cyco130** [comment]
Thanks for reporting. It appears to be an issue with Vite 3.0.1 and 3.0.2. You can change the dependency from `"vite": "^3.0.0"` to `"vite": "3.0.0"` and reinstall as a temporary workaround while I figure out what the problem is.

**Comment by cyco130** [comment]
Fixed in 1.4.1.
